fractal_tasks_core/dev/lib_args_schemas.py

```diff
@@ -78,49 +78,6 @@ def _remove_pydantic_internals(old_schema: _Schema) -> _Schema:
     return new_schema
 
 
-def _get_args_descriptions(executable) -> dict[str, str]:
-    """
-    Extract argument descriptions for a task function
-    """
-    # Read docstring (via ast)
-    module_path = Path(fractal_tasks_core.__file__).parent / executable
-    module_name = module_path.with_suffix("").name
-    tree = ast.parse(module_path.read_text())
-    function = next(
-        f
-        for f in ast.walk(tree)
-        if (isinstance(f, ast.FunctionDef) and f.name == module_name)
-    )
-    docstring = ast.get_docstring(function)
-    # Parse docstring (via docstring_parser) and prepare output
-    parsed_docstring = parse(docstring)
-    descriptions = {
-        param.arg_name: param.description.replace("\n", " ")
-        for param in parsed_docstring.params
-    }
-    return descriptions
-
-
-def _include_args_descriptions_in_schema(*, schema, descriptions):
-    """
-    Merge the descriptions obtained via `_get_args_descriptions` into an
-    existing JSON Schema for task arguments
-    """
-    new_schema = schema.copy()
-    new_properties = schema["properties"].copy()
-    for key, value in schema["properties"].items():
-        if "description" in value:
-            raise ValueError("Property already has description")
-        else:
-            if key in descriptions:
-                value["description"] = descriptions[key]
-            else:
-                value["description"] = "Missing description"
-            new_properties[key] = value
-    new_schema["properties"] = new_properties
-    return new_schema
-
-
 def create_schema_for_single_task(
```

Comment:
Updates to create_schema_for_single_task

After schema = vf.model.schema(), and after the `_remove_*` functions, proceed as follows.

0. Extract pkg_name (i.e. package), module_relative_path (=executable), and function_name (Path(executable).with_suffix("").name).
1. Get function-args descriptions (new function)
2. Insert function-args descriptions (new function)
3. Iterate over hard-coded list of models and relatives paths: get and insert descriptions for each model (with new functions).
4. In the create_schema_for_single_task signature, include a flag `include_descriptions=True/False`.

Reply:
> In the create_schema_for_single_task signature, include a flag include_descriptions=True/False.

Not needed, as we can pass `inner_pydantic_models={}`